assets/css/abstracts/_variables.scss

```diff
@@ -1,14 +1,12 @@
 @import "node_modules/@wordpress/base-styles/variables";
 
-// @todo replace with $grid-unit values when `@woocommerce/base-styles` is updated.
-// see: https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_variables.scss#L23-L36
-$gap-largest: 40px; // $grid-unit-50
-$gap-larger: 36px;
-$gap-large: 24px; // $grid-unit-30
-$gap: 16px; // $grid-unit-20
-$gap-small: 12px; // $grid-unit-15
-$gap-smaller: 8px; // $grid-unit-10
-$gap-smallest: 4px; // $grid-unit-05
+$gap-largest: $grid-unit-50;
+$gap-larger: 4.5 * $grid-unit;
+$gap-large: $grid-unit-30;
+$gap: $grid-unit-20;
+$gap-small: $grid-unit-15;
+$gap-smaller: $grid-unit-10;
+$gap-smallest: $grid-unit-05;
```

Comment:
Instead of hardcoding px values, let's use the `$grid-unit` variables from `@wordpress/components`.